kroxylicious/src/main/java/io/kroxylicious/proxy/KafkaProxy.java

```diff
@@ -201,6 +202,11 @@ else if (KQueue.isAvailable()) {
             bindFuture = serverBootstrap.bind(proxyPort);
         }
         acceptorChannel = bindFuture.sync().channel();
+        // Pre-register counters to avoid creating them on first request and thus skewing the request latency
+        Metrics.counter("inbound_downstream_messages");
```

Comment:
we should have constants for the metric names.

Reply:
Hello Keith, is what I've amended my commit with what you were thinking? Should they be local variables in the method instead, or is this fine?

Reply:
I'm wondering if it would be worth having a class like `KroxyliciousMetrics` with the counters defined as constants and you could refer to them from both places. So the class might have methods like `static Summary getRequestSizeSummary()`, or offer public static fields. It could help you discover what metrics are out there too.

Reply:
How do others feel about a central class defining all metrics? It's possibly easier from a docs/operator perspective but there is no way to ensure it's exhaustive and would be easily forgotten when adding/amending metrics. 

I wonder if we could somehow scrape the source or dump the prometheus output after a test run to catch the set of metrics we are actively using (that does assume a comprehensive test suite tho)